src/app/App.tsx

```diff
@@ -47,7 +45,16 @@ class App extends React.Component<{}> {
     render() {
         return (
             <PersonContext.Provider
-                value={(store.getState().restEndepunkter.personinformasjon.data as Person).fødselsnummer || undefined}
+                value={
```

Comment:
Dette kan kanskje skrives på en penere måte?

Reply:
Haha, ventet på den...Åpen for forslag. Ville bare ha ting til å funke